buckets.yaml

```diff
@@ -7,11 +8,15 @@
 #     test-history collection phase, and defaults to true.
 
 gs://kubernetes-jenkins/logs/:
+  contact: "fejta"
   prefix: ""
 gs://kube_azure_log/:
+  contact: "travisnewhouse"
   prefix: "azure:"
-gs://rktnetes-jenkins/logs/:
+gs://rktnetes-jenkins/logs/:  # @euank
```

Comment:
strip the comment too for consistency.
The key could also be `contact_github_username` or such, though just specifying it in the comment at the top seems cleaner.